substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -1296,21 +1296,26 @@ pub mod pallet {
 				Error::<T>::TooManyTargets
 			);
 
+			let mut targets = targets
+				.into_iter()
+				.map(|t| T::Lookup::lookup(t).map_err(DispatchError::from))
```

Comment:
Could we use `AccountId` here? I'm not sure why we are using the `Lookup` type in this context.